lib/vmpooler/api/helpers.rb

```diff
@@ -158,6 +160,10 @@ def get_tag_metrics(backend, date_str)
             hostname, tag = key.split(':', 2)
           end
 
+          if opts[:only]
+            next unless tag == opts[:only]
```

Comment:
Above, `:only` is a boolean (`:only => false`) and here it is being compared to a string (`tag = 'unknown').